zeek/assets/logs/zeek.yaml

```diff
@@ -345,17 +413,30 @@ pipeline:
     - type: service-remapper
       name: Define `_path` as the official service of the log
       enabled: true
-      meta:
-        tags: []
       sources:
         - _path
-    - type: date-remapper
-      name: Define `ts` as the official date of the log
+    - type: pipeline
```

Comment:
Also here we need an integration test to cover this case

Reply:
Updated